CONTRIBUTING.md

```diff
@@ -103,7 +98,7 @@ $ git add <list of modified files>
 Now, commit these changes using
 
 ```
-$ git commit -s
+$ git commit -s -a
```

Comment:
Its good to create a new branch and commit the changes over there. For example,
git checkout -b new_branchname

Create a new remote for the gluster/glusto-tests repo with the command,
git remote add fork https://github.com/gluster/glusto-tests.git

Commit your changes using, 
git add <files.py>
git commit -s -m "Comments for the commit" <files.py>

Reply:
Please have a look at the complete file it's already present hence haven't added it.

Reply:
I have gone through the complete file and then added this comment. Would be better if we could re-order the steps